scripts/manage_tx.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+import configparser
+import json
+import os
+import re
+from argparse import ArgumentParser
+from pathlib import Path
+
+from transifex.api import transifex_api
+
+ORG_SLUG = "python-doc"
+PROJ_SLUG = "python-newest"
+
+
+def _get_tx_token() -> str:
+    """
+    Try to get Transifex API token from some sources.
+    First try TX_TOKEN environment variable, then ~/.transifexrc file.
+    """
+    token = ""
+    if 'TX_TOKEN' in os.environ:
+        token = os.environ['TX_TOKEN']
+    else:
+        config = configparser.ConfigParser()
+        config.read(f"{os.path.expanduser('~')}/.transifexrc")
+        for section in config.sections():
+            if section in ["https://www.transifex.com", "https://app.transifex.com"]:
+                token = config[section]["token"]
+                break
+    return token
```

Comment:
```suggestion
def _get_tx_token() -> str:
    """
    Try to get Transifex API token from some sources.
    First try TX_TOKEN environment variable, then ~/.transifexrc file.
    """
    if os.path.exists(".tx/api-key"):
        with open(".tx/api-key") as f:
            return f.read()

    config = configparser.ConfigParser()
    config.read(os.path.expanduser("~/.transifexrc"))
    try:
        return config["https://www.transifex.com"]["token"]
    except KeyError:
        pass

    return os.getenv("TX_TOKEN", "")
```
Function from sample workflows script, https://github.com/python-docs-translations/transifex-automations/blob/c1bc8b3317ecf0f35f45b6c42bd5ad36c37e4f90/sample-workflows/transifex-util.py#L94 to check `.tx/api-key` too.

Reply:
Thanks. I think I'll add .tx/api-key, but also keep TX_TOKEN as a first option. TX_TOKEN is the official environment variable for Transifex CLI tool, so it requires as API token.